docs/outlook/debug-ui-less.md

```diff
@@ -26,11 +29,11 @@ Otherwise, if you used another tool to create your add-in, perform the following
 
 ## Configure and run the debugger
 
-Now that you've enabled debugging on your add-in, you're ready to configure and run the debugger. For instructions on how to do this, select one of the following options that applies to your runtime.
+Now that you've enabled debugging on your add-in, you're ready to configure and run the debugger. For instructions on how to do this, select one of the following options that applies to your webview control. (For information about how to determine what webview control is used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).)
```

Comment:
Not sure the parentheses are needed, since this follows the previous sentence as a complete thought.
```suggestion
Now that you've enabled debugging on your add-in, you're ready to configure and run the debugger. For instructions on how to do this, select one of the following options that applies to your webview control. For information about how to determine what webview control is used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```